docs/bootstrap-application.md

```diff
@@ -0,0 +1,109 @@
+# Sparkle Web App
+
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
+
+### Connect your application to Firebase
+
+<!-- section 1 -->
+
+## 'Bootstrap' the platform
+
+Make sure that you're located in ./scripts folder.
+
+First of all you'd need to boostrap an environment using the following command:
+
+* Run the bootstrap script `./bootstrap-new-environment.ts TODO-PROJECT-ID [PRIVATE-KEY-FILE].json`
+
+As a result you should be able to see the 'Bootstraping complete' message
+<!--<!-- ![image](./images/creating-new-environments/57-bootstrap-complete.png) --> -->
+
+Then we'll be able to add a user to your application:
+
+```
+⇒ ./create-users.ts [PRIVATE-KEY-FILE].json user@sparkle.space
+
+User created: email=user@sparkle.space, password=REDACTED
+
+```
+
+We seem to need to manually create the `roles` collection in firestore, with a single `admin` document, that contains the following fields:
+
+* `allowAll` (boolean): `false`
+* `users` (array of strings): either an empty array (`[]`), or adding the user uid's copied from Firebase Authentication for the users who need to be admins.
+
+**Note:** you also need to be added to the `owners` field on a venue to be able to edit it. The `roles` collection just controls whether you're able to access the admin panel at all.
+
+<!-- ![image](./images/creating-new-environments/53-roles-collection.png) -->
+
+Once complete you should be able to make that user an admin. To do so run this command:
+* `./update-admin-role-users.ts [PRIVATE-KEY-FILE].json ADD user@sparkle.space`
+
+As a result you'll see this message:
+<!-- ![image](./images/creating-new-environments/58-user-admin-role.png) -->
+
+<!-- section 2 -->
+
+## Configure ENV file
+
+In order for your application to connect to the proper Firebase environment you'd need to set up .env.local config. Please follow steps below:
+
+* Locate or create .env.local file inside project root folder
+* Paste the following config
+`
+REACT_APP_PROJECT_ID=
+REACT_APP_API_KEY=
+REACT_APP_APP_ID=
+REACT_APP_MEASUREMENT_ID=
+REACT_APP_BUCKET_URL=
+`
+
+Here's a list of places where you can find the config values:
+* REACT_APP_PROJECT_ID, REACT_APP_APP_ID and REACT_APP_API_KEY can be found in Firebase - Project Settings - General tab
+* REACT_APP_MEASUREMENT_ID can be found at https://analytics.google.com/analytics/web. You can search for Measurement ID in the search bar at the top of the page
+* REACT_APP_BUCKET_URL con ofsists REACT_APP_PROJECT_ID and equals REACT_APP_PROJECT_ID.appspot.com
```

Comment:
```suggestion
You can find all of these values in the Firebase - Project Settings - General tab.
```

Reply:
agree to remove everything except the measurement_id, that one cannot be located at firebase, so let's leave that info here

Reply:
025e814